superset/config.py

```diff
@@ -705,3 +705,9 @@ class CeleryConfig(object):
     except Exception:
         logging.exception("Found but failed to import local superset_config")
         raise
+
+
+# Note currently SIP-15 feature is WIP and should not be enabled.
+SIP_15_ENABLED = False
```

Comment:
i think this should all be above line 682, otherwise you'll be overriding custom configs here